### ISSUE TITLE
Lots of problems importing Lutris library created with former ppa application

### ISSUE THREAD

**Issue by H34dl3ss** [lutris/lutris#4793]
Hi, I used the former ppa version of Lutris with Linux Mint 20 and configured several games on 3 PCs.
While upgrading to Mint 21 the ppa had to be removed along with the Lutris app.
Library and settings are still there though.

Now I installed the version from flatpak and it gives me headaches.

Issue 1:
I chose mass detect installed games.
It found 2, which are working fine. The rest of games is shown in the lower section of that "detection" screen and they are not "imported".

Issue2:
I installed the games to `/games`.
When browsing to /games and choosing the folder it will change the path to something like `/run/user/1001/whatever/some_cryptic_id`
Why is this?
I tried to get some of the games running that were not detected properly. Of course nothing is working any more :(
After some trials, those directories with "cryptic_ids" have multiplied and eat up significant disk space.
What is going on?
How are those directories related?

Issue3:
I tried installing a Windows game and after entering `/games` as destination it complained about write permissions.
I checked the directory and rights are set for the logged in user. Creating directories in file explorer is working.
Ignoring the warning starts the install, but i can't see a new directory being created.
Eventually the process crashes complaining about memory shortage.
I tried the same with browsing to `/games` which lead to the mentioned `/run/user/1001/whatever` with the same result.
Where are the files created?

Q:
1. Is there a documentation where lutris is installing files?
2. Why is the chosen destination folder altered to `/run/user/whatever`
3. How can I port my existing library without having to install everything from scratch?
4. Is the flatpak edition recommended or should I use the LM repository although the version is rather outdated?

It's frustrating - I'd really appreciate some help.
It would be a real bummer to have to set up each library from scratch on 3 PCs.

**Comment by danieljohnson2** [comment]
Are you sure you are giving us the correct paths here? All your paths are right at the filesystem root; they should be things like /home/me/games/ or /home/me/.var/app/something/something, I think.

Flatpak application installs are per-user, not system-wide, or so I thought.

**Comment by H34dl3ss** [comment]
In fact, when I started with Lutris, I set the default installation folder to `/games` because I wanted to have the data available for all users on that PC.
If i select the default installation folder via the browse button it looks like this:
![image](https://user-images.githubusercontent.com/57181552/229291215-25c4c491-193c-45a1-b034-1b8c93615b15.png)

The content of /run/user/1001/doc
![image](https://user-images.githubusercontent.com/57181552/229291775-cc0d9a10-5a46-4a1b-8ff4-7258c7cd5d74.png)

**Comment by danieljohnson2** [comment]
I'm pretty sure that this is not going to work; Lutris wants to do things like write files into game directories, and Flatpak is supposed to be sandboxed so it will restrict further what Lutris gets to do.

You're going to have to put the games into your home directory somewhere.

**Comment by H34dl3ss** [comment]
It's just a mess!
I played around, trying to understand what's going on.
First I thought the folders would get hardlinked, date and time as well as count indicated so.
Tried to verify the inodes, but those were not identical.
Then I decided to delete one of those non working folders and well...it deleted the source in /games as well. Must have been hardlinked after all.

I guess I will have to uninstall the flatpak and see if the deb version is easier to handle. There is no way I can live with such a sophisticated design. Troubleshooting is hell on earth like this.

Given the fact, that games are tens of gigabyte in size, why does every user need it's own copy in his home directory?
In a multi user environment this approach will eat up space without need, but this is a different story.
